config/routes.rb

```diff
@@ -12,5 +12,6 @@
     get "/login", to: "sessions#new"
     post "/login", to: "sessions#create"
     get "/logout", to: "sessions#destroy"
+
```

Comment:
you missing define additional resources for user actions (index, edit, update, destroy) at line 10